models/baseModels/Invoice/Invoice.ts

```diff
@@ -166,29 +166,31 @@ export abstract class Invoice extends Transactional {
     await super.afterSubmit();
 
     // update outstanding amounts
-    await this.fyo.db.update(this.schemaName, {
-      name: this.name as string,
-      outstandingAmount: this.baseGrandTotal!,
-    });
+    if (this.schemaName !== 'SalesQuote') {
```

Comment:
use early exits. also don't hardcode model values like this use `ModelNameEnum` for similar use cases